### PR TITLE
Cli create AWS secret

### PR DESCRIPTION
**Pull request by marcogschmidt**
Added `supergloo create secret aws` command to CLI. We need this in order for supergloo to be able to manage AWS App Mesh resources.

**NOTE**: check the first commit for the actual changes.

### CODE REVIEW

File: cli/pkg/options/options.go [hunk 1]
```diff
@@ -81,6 +82,13 @@ type CreateTlsSecret struct {
 	CaCertFilename     string
 }
 
+type AwsSecret struct {
```
Comment (ilackarms) [inline cli/pkg/options/options.go]: nit: CreateAwsSecret

Comment (marcogschmidt) [reply cli/pkg/options/options.go]: Renamed

**Pull request merged**